CONTRIBUTING.md

```diff
@@ -0,0 +1,190 @@
+💖:tada: Thanks for taking the time to contribute to GATK-SV! :tada:💖
+
+When it comes to open-source, every contribution makes the 
+software better for everyone and is warmly appreciated by the 
+community. To express our gratitude for your contribution, 
+we would like to provide you with easy-to-follow steps to get started.
+
+# Code of Conduct
+
+In the interest of fostering an open and welcoming environment, we as 
+contributors and maintainers pledge to make participation in our 
+project and our community a harassment-free experience for everyone, 
+regardless of age, body size, disability, ethnicity, sex characteristics, 
+gender identity and expression, level of experience, education, 
+socio-economic status, nationality, personal appearance, race, religion, 
+or sexual identity and orientation. 
+
+GATK-SV and everyone participating in it is governed by the 
+[Broad Institute Code of Conduct](https://github.com/broadinstitute/.github/blob/develop/CODE_OF_CONDUCT.md).
+By participating, you are expected to uphold this code.
+
+
+# What Should I Know Before I Get Started?
+
+GATK-SV is a _pipeline_ written in 
+[Workflow Description Language](https://openwdl.org/) (WDL) 
+that orchestrates joint genotyping structural variations using 
+open-source tools and custom scripts. The tools and scripts are distributed 
+in Docker containers, and the pipeline is executed using a 
+[Cromwell](https://cromwell.readthedocs.io/en/stable/) 
+server running on commercial cloud platforms. 
+You may interface with a Cromwell server using 
+[Cromshell](https://github.com/broadinstitute/cromshell) or 
+[Terra platform](https://terra.bio). We currently publish 
+Docker images on Google Container Registry (GCR) and Azure 
+Container Registry (work-in-progress).
```

Comment:
```suggestion 
```
I think we should save this kind of info for the README, to keep things more consolidated

Reply:
Agree!